tests/show_untested_options.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# This file MUST remain co-located with the files to work.
+
+
+# Find paths to the tests and the dune executable.
+SCRIPT=`readlink -f "$0"`
+TEST_DIR=`dirname "$SCRIPT"`
+DUNE_DIR=`dirname "$TEST_DIR"`
+DUNE="$DUNE_DIR/dune"
+
+# Get a list of the options.
+options=`$DUNE --help | grep -o "^  --[a-z\-]*"`
+
+# Get a list of the test files.
+files=`find "$TEST_DIR" | grep "[.]py\$"`
+
+# Return value. Initially set to zero/success.
+rv=0
+
+# Search for each option.
+for opt in $options; do
+    if ! grep --quiet \"$opt\" $files ; then
+        # Report missing options and set the return value to non-zeor/failure.
+        echo "Missing option: $opt"
```

Comment:
I have built DUNE on your branch and I get:
E               subprocess.CalledProcessError: Command '/home/mikel/repo/DUNE/tests/show_untested_options.sh' returned non-zero exit status 1.

/usr/lib/python3.8/subprocess.py:516: CalledProcessError
--------------------------------------------------------- Captured stdout call ---------------------------------------------------------
Missing option: --bootstrap-system
Missing option: --get-table
Missing option: --activate-feature

Maybe we should remove this test from this PR and create a separate issue for it to add missing options?

Reply:
The test itself should stay. I believe it fails correctly.

None the less, if we believe - as a group - it's better to bring the tests to 100% pass and write issues against them, then we should add the flags to `tests/untested.py` to "find" the flag.

Reply:
IMO it's easier to have 100% passing and tests executed after every merge / commit (executing them would be a separate issue). If something fails we are notified and we can fix the problem.
If you agree can you add those flags to untested.py?

Reply:
I have created a new issue to have 100% tests passing and run functional tests every PR / merge:
https://github.com/AntelopeIO/DUNE/issues/86